addons/old-studio-layout/addon.json

```diff
@@ -56,14 +56,18 @@
       "url": "default.css",
       "matches": ["https://scratch.mit.edu/studios/*"],
       "if": {
-        "settings": { "version": "default" }
+        "settings": {
+          "version": "default"
+        }
       }
     },
     {
       "url": "scratchr2.css",
       "matches": ["https://scratch.mit.edu/studios/*"],
       "if": {
-        "settings": { "version": "scratchr2" }
+        "settings": {
+          "version": "scratchr2"
+        }
```

Comment:
Looks like your code editor decided to change the formatting. This probably wasn't intentional.